src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -190,10 +190,12 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
     // Return the allowable values for readFilterNames/disableReadFilter
     @Override
     public Set<String> getAllowedValuesForDescriptorArgument(final String longArgName) {
-        if (longArgName.equals(StandardArgumentDefinitions.READ_FILTER_LONG_NAME) ||
-                longArgName.equals(StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME)) {
+        if (longArgName.equals(StandardArgumentDefinitions.READ_FILTER_LONG_NAME)) {
             return readFilters.keySet();
         }
+        if (longArgName.equals(StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME)) {
+            return toolDefaultReadFilters.keySet();
+        }
```

Comment:
For disable filters the test is easy and I will do it. For the other one, it is a bit tricky: it returns all the read filters implemented and it could break whenever a new one is included into the package. Any idea on how to do it without the need of change every time a new `ReadFilter` in the package is added?